.projenrc.ts

```diff
@@ -27,7 +27,7 @@ import {
 const GITHUB_USER = 'awslabs';
 const PUBLICATION_NAMESPACE = 'cdklabs';
 const PROJECT_NAME = 'generative-ai-cdk-constructs';
-const CDK_VERSION: string = '2.111.0';
+const CDK_VERSION: string = '2.117.0';
```

Comment:
In the process of testing `2.85.0` which is the lowest for `aws-cdk-lib` without an `npm audit` known vulnerability.

Reply:
version `2.84.0` has some security vulnerabilities and `2.85.0` currently doesn't:

```
% export CDK_VERSION=2.84.0; npm install aws-cdk-lib@${CDK_VERSION} && npm list  && npm uninstall aws-cdk-lib && npm list

added 5 packages, and audited 40 packages in 2s

3 packages are looking for funding
  run `npm fund` for details

2 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix --force

Run `npm audit` for details.
temp@ /Users/scottschreckengaust/temp
└── aws-cdk-lib@2.84.0


removed 39 packages, and audited 1 package in 641ms

found 0 vulnerabilities
/Users/scottschreckengaust/temp
└── (empty)

% export CDK_VERSION=2.85.0; npm install aws-cdk-lib@${CDK_VERSION} && npm list  && npm uninstall aws-cdk-lib && npm list

added 5 packages, and audited 40 packages in 1s

3 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
temp@ /Users/scottschreckengaust/temp
└── aws-cdk-lib@2.85.0


removed 39 packages, and audited 1 package in 645ms

found 0 vulnerabilities
/Users/scottschreckengaust/temp
└── (empty)
```

Reply:
Lowest version that builds is `2.88.0`.

Reply:
Version lowered to 2.88.0